internal/gonut/cf/report.go

```diff
@@ -21,9 +21,21 @@
 package cf
 
 import (
+	"encoding/json"
 	"fmt"
 	"strings"
 	"time"
+
+	"gopkg.in/yaml.v2"
+)
+
+//OutputType report output type
```

Comment:
Really tiny thing, usually I like to have a very common style. Could you add a space between the `//` and the start of the comment?